src/Microsoft.Fx.Portability/ObjectModel/CompatibilityRangeAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+#if FEATURE_SERIALIZABLE
```

Comment:
We're using this type in the service to load information about the analyzers, which currently requires us to load in a separate app domain, necessitating the [Serializable] to send the information back